runtime/src/serde_snapshot.rs

```diff
@@ -735,27 +762,10 @@ impl<'a> Serialize for SerializableAccountsDb<'a> {
             )
         }));
         let slot = self.slot;
-        let accounts_delta_hash = self
```

Comment:
Can we collect "version" in ABS too? 
```
        let version = self.accounts_db.write_version.load(Ordering::Acquire);
```

This way we can avoid the dependency of "AccountsDB" in the `serialize`?